src/frontend/scss/components/templates/courses/cms/_course_detail.scss

```
@@ -131,6 +131,34 @@
         }
       }
     }
+
+    // Add CSS style only to edit mode
+    &__hidden {
+      h3.cms-render-model {
+        &::before {
+          background-image: r-theme-val(course-detail, run-catalog-visibility-hidden-logo);
+          content: '';
+          background-size: 1rem 1rem;
+          display: inline-block;
+          width: 1rem;
+          height: 1rem;
+          margin-left: -1rem;
+        }
+      }
+    }
+    &__course_only {
+      h3.cms-render-model {
+        &::before {
+          background-image: r-theme-val(course-detail, run-catalog-visibility-course-only-logo);
+          content: '';
+          background-size: 1rem 1rem;
+          display: inline-block;
+          width: 1rem;
+          height: 1rem;
+          margin-left: -1rem;
+        }
+      }
+    }
```

Group common styles to ease maintainability.
Furthermore to respect BEM syntax, this is modifier, so use `--` instead of `__` (e.g: ~`&__header`~ `&--header`)

```suggestion
    &--hidden,
    &--course_only {
      h3.cms-render-model::before {
        background-size: 1rem 1rem;
        content: '';
        display: inline-block;
        height: 1em;
        margin-left: -1rem;
        width: 1em;
        }
    }

    &--hidden {
      h3.cms-render-model::before {
          background-image: r-theme-val(course-detail, run-catalog-visibility-hidden-logo);
      }
    }

    &--course_only {
      h3.cms-render-model::before {
          background-image: r-theme-val(course-detail, run-catalog-visibility-course-only-logo);
      }
    }
```

---

Right! Changed.